src/pages/extending-chat-widget/chat-widget-moments/index.mdx

```diff
@@ -184,6 +185,24 @@ This method has no parameters.
 momentsSDK.close();
 ```
 
+### setIsFragile
+
+Set `isFragile` value. It can be provided in initalization and changed later with this function.
+Parameters:
+
+-   `value` - Boolean
+
+```js
+momentsSDK.setIsFragile(true)
+```
+
+## LauncherData
+
+Data passed from the Chat Widget. It is available since the moment of SDK initialization. It consists of:
```

Comment:
using the term "moment" is a little bit unfortunate in the context of this feature 😅 
maybe something like those will also suit it well?
```suggestion
Data passed from the Chat Widget. It is available right after the SDK initialization. It consists of:
```